lib/hooks/useComponentDragging.ts

```diff
@@ -97,16 +94,16 @@ export const useComponentDragging = ({
         edit_event_id: Math.random().toString(36).substr(2, 9),
         edit_event_type: "edit_schematic_component_location",
         schematic_component_id: schematic_component_id,
-        original_center,
-        new_center: { ...original_center },
+        original_center: current_position,
+        new_center: { ...current_position },
         in_progress: true,
         created_at: Date.now(),
         _element: componentGroup as any,
       }
 
       activeEditEventRef.current = newEditEvent
     },
-    [cancelDrag, enabled, circuitJson],
+    [cancelDrag, enabled, circuitJson, editEvents],
   )
```

Comment:
this is the only functional change here

Reply:
Now the centre is getting updated as per the last position of the moved component